engine/examples/client/pike/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/libsv/go-bk/bec"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/pike"
+	"github.com/bitcoin-sv/spv-wallet/engine/script/template"
+)
+
+func main() {
```

Comment:
maybe it's good idea to change this into [testable example](https://go.dev/blog/examples) and put in the pike package?
We could even use the `Output` feature of testable examples to check the result of the method

Reply:
done